packages/expo-sqlite/ios/SQLiteModuleNext.swift

```diff
@@ -204,6 +204,30 @@ public final class SQLiteModuleNext: Module {
     return directory?.appendingPathComponent(name)
   }
 
+  private func convertToUrl(string value: String) -> URL? {
```

Comment:
Can't we reuse the function from FileSystem utilities or another one from the Convertible implementation of URL?